app/javascript/packs/application.js

```diff
@@ -26,12 +26,17 @@ import "bootstrap";
 
 // Internal imports, e.g:
 import { initFlatpickr } from "../plugins/flatpickr";
+import { initFlatRange } from "../plugins/flatpickr";
 import { initChatRoomCable } from "../channels/chat_room_channel";
 import { initAutocomplete } from "../plugins/init_autocomplete";
 
 document.addEventListener('turbolinks:load', () => {
   // Call your functions here, e.g:
-  initFlatpickr();
+  initFlatpickr("#month_start", {
```

Comment:
Remember not to put this methods in the application.js, makes it a little messy.